src/messaging/ReliableMessageContext.h

```diff
@@ -164,6 +168,9 @@ class ReliableMessageContext
 
         /// When set, signifies that the exchange created sorely for replying a StandaloneAck
         kFlagEphemeralExchange = (1u << 9),
+
+        /// When set, automatically release the session when finishing its work. Used by UpdateNOC command
```

Comment:
```suggestion
        /// When set, automatically release the session when this exchange is destroyed.
```
"its" is ambiguous otherwise....